desk/sur/treaty.hoon

```diff
@@ -8,7 +8,7 @@
   ::    Includes the docket itself, plus provenance generated by clay
   ::
   +$  treaty
-    [=ship =desk =case hash=@uv =docket:docket]
+    [=ship =desk vips=crew:clay =case hash=@uv =docket:docket]
```

Comment:
I think you want something like 

```hoon
+$  permission  ?(%open %allowed %not-allowed)
::
+$  treaty
  [=ship =desk =permission =case hash=@uv =docket:docket]
```

so we don't leak who's actually permissioned to the desk to random requesters